aikau/src/main/resources/alfresco/services/DialogService.js

```diff
@@ -661,6 +668,25 @@ define(["dojo/_base/declare",
       },
 
       /**
+       * Handles failed form submission by re-enabling the dialog buttons so that the user to submit
+       * alternative data.
+       * 
+       * @instance
+       * @param {object} dialog The dialog to enable the buttons for
+       * @since 1.0.58
+       */
+      onFailedSubmission: function alfresco_services_DialogService___onFailedSubmission(dialog) {
+         if (dialog)
+         {
+            when(dialog.getButtons(), lang.hitch(this, function(buttons) {
+               array.forEach(buttons, function(button) {
+                  button.set("disabled", false);
```

Comment:
You can't submit a form if it has invalid controls. Therefore this scenario won't occur unless the developer intentionally provides a way to publish the enablement topic from within the dialog (because nothing else will be accessible on the screen). This would be quite a foolish thing for them to do, and we can't code against self-destructive behaviour.